src/windows/editor_object_selection.c

```diff
@@ -903,6 +912,8 @@ static void window_editor_object_selection_resize(rct_window *w)
 	window_set_resize(w, 600, 400, 1200, 1000);
 }
 
+#define FILTER_DROPDOWN_POSITION_SELECTED 5
```

Comment:
Okay, that makes more sense cause the original had no index names and i was thinking "why define these? they're like 5 lines above? wouldn't it be weird for just 2 to be defined and none other? but i guess..." since the pre-modified had no names and i remember the menu button dropdowns having no names, but now that you say make a new enum for the entire index that makes much more sense. i'll do a CTRL+F for DDIX to make sure it matches the others as well lol.I'll work on this when I get back later this week. Thanks!